sdk/storage/Azure.Storage.DataMovement/KnownIssues.md

```diff
@@ -26,4 +26,40 @@ The Azure Storage REST service has various limitations as to the size of request
 
 ### Resume Behavior when Client Options are set
 
-When calling `*StorageResourceProvider.FromClient` with a Storage Client (e.g. `BlobBaseClient`, `ShareFileClient`) initialized with `*ClientOptions` (e.g `BlobClientOptions`, `ShareClientOptions`). It is NOT guaranteed that properties set within the `*ClientOptions` will be respected when resuming a transfer. 
\ No newline at end of file
+When calling `*StorageResourceProvider.FromClient` with a Storage Client (e.g. `BlobBaseClient`, `ShareFileClient`) initialized with `*ClientOptions` (e.g `BlobClientOptions`, `ShareClientOptions`). It is NOT guaranteed that properties set within the `*ClientOptions` will be respected when resuming a transfer.
+
+### Encryption Scope Support
+
+There is currently no support to specify an encryption scope for operations such as uploads, downloads, or copies.
+
+### Recursive Directory Support
+
+There is currently no support to handle operations that involve processing an entire directory, including all its subdirectories and files, automatically in a single command.
+
+### Blob to File with OAuth/Entra ID
+
+There is currently no support for transferring data from an Azure Blob Storage to a file system using authentication based on Microsoft Entra ID.
+
+### Resume Transfers started from any Beta version
```

Comment:
This will sound silly.. we should also add that you cannot resume from transfers from the deprecated Microsoft.Azure.Storage.DataMovement package.
(I have a feeling someone will ask)